GDJS/Runtime/events-tools/commontools.ts

```diff
@@ -371,3 +371,18 @@ namespace gdjs {
     }
   }
 }
+
+      /**
+       * Linearly interpolates between two angles (in degrees) by taking the shortest direction.
+       * @param angle1 Starting angle, in degrees.
+       * @param angle2 Destination angle, in degrees.
+       * @param x The interpolation value between 0 and 1.
+       * @returns Return the interpolated angle, in degrees.
+       */
+       export const lerp_angle = function (
+        angle1: number,
+        angle2: number,
+        x: float
+      ): number {
```

Comment:
In typescript (javascript), `number`, `float` and `integer` are the same.
It can help a human understand some things though. Here I would use `float` as return type since `x` is supposed to be a float.